recipes/folly/all/conanfile.py

```diff
@@ -23,54 +22,60 @@ class FollyConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
-        "use_sse4_2" : [True, False],
+        "use_sse4_2": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
-        "use_sse4_2" : False
+        "use_sse4_2": False
     }
 
-    generators = "cmake", "cmake_find_package"
-
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _minimum_cpp_standard(self):
+    def _min_cppstd(self):
         return 17 if Version(self.version) >= "2022.01.31.00" else 14
 
     @property
-    def _minimum_compilers_version(self):
+    def _compilers_minimum_version(self):
         return {
             "Visual Studio": "15",
             "gcc": "5",
             "clang": "6",
             "apple-clang": "8",
-        } if self._minimum_cpp_standard == 14 else {
+        } if self._min_cppstd == 14 else {
             "gcc": "7",
             "Visual Studio": "16",
             "clang": "6",
             "apple-clang": "10",
         }
 
-    def export_sources(self):
-        self.copy("CMakeLists.txt")
-        for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            self.copy(patch["patch_file"])
+    def _cppstd_less_than(self, cppstd, min_cppstd):
```

Comment:
This is wonky... the compiler version checks should be taking care of this (making sure the assumed default is high enough) 

It's the only piece I am iffy about 🤔 

If we assume
- the min allowed compiler
- high enough default cppstd

do we need to manually check that it's actually true?

Reply:
Removed default cppstd checks. I will follow the current conventions. Additionally if the user doesn't provide `compiler.cppstd` then recipe will use _min_cppstd at `CXX_STD/MSVC_LANGUAGE_VERSION`.